index.js

```diff
@@ -17,6 +22,10 @@ module.exports = function () {
 
 		try {
 			file.contents = new Buffer(gcmq(file.contents.toString()));
+			// apply source map to the chain
+			if (file.sourceMap) {
+				applySourceMap(file, result.map);
```

Comment:
`result` is undefined and not used anywhere else.

Reply:
Maybe adding a linter might help (?)

Do you want me to add one?

Reply:
Feel free to add `eslint` in another PR.